src/mapper.rs

```diff
@@ -173,10 +173,10 @@ impl<'s> ProguardMapper<'s> {
     /// # Examples
     ///
     /// ```
-    /// use proguard::ProguardMapper;
+    /// use proguard::Mapper;
     ///
     /// let mapping = r#"android.arch.core.executor.ArchTaskExecutor -> a.a.a.a.c:"#;
-    /// let mapper = ProguardMapper::from(mapping);
+    /// let mapper = Mapper::from(mapping);
```

Comment:
Since you asked for bikeshedding; this is probably nicer as `let mapper = proguard::Mapper` without the `use`.